specification/resources/resource-manager/readme.md

```diff
@@ -81,6 +81,74 @@ tag: package-changes-2022-05
 tag: package-snapshots-2022-11
 ```
 
+### Tag: package-policy-2023-04
+
+These settings apply only when `--tag=package-policy-2023-04` is specified on the command line.
+
+``` yaml $(tag) == 'package-policy-2023-04'
+input-file:
+- Microsoft.Authorization/stable/2020-09-01/dataPolicyManifests.json
+- Microsoft.Authorization/stable/2023-04-01/policyDefinitions.json
+- Microsoft.Authorization/stable/2023-04-01/policyDefinitionVersions.json
+- Microsoft.Authorization/stable/2023-04-01/policySetDefinitions.json
+- Microsoft.Authorization/stable/2023-04-01/policySetDefinitionVersions.json
+- Microsoft.Authorization/stable/2023-04-01/policyAssignments.json
+- Microsoft.Authorization/preview/2022-07-01-preview/policyExemptions.json
+- Microsoft.Authorization/preview/2022-08-01-preview/policyVariables.json
+- Microsoft.Authorization/preview/2022-08-01-preview/policyVariableValues.json
```

Comment:
One tag shall only contain one api-version, otherwise it will cause Python SDK generation failure.

Reply:
Can you elaborate?  This tag has a single api-version for each type.

Reply:
Your new tag `package-policy-2023-04` contains multi api-version `2020-09-01/2023-04-01/...`:
![image](https://github.com/Azure/azure-rest-api-specs/assets/70930885/fd2ecdc6-9144-4601-8cd1-642aaef1132a)

![image](https://github.com/Azure/azure-rest-api-specs/assets/70930885/e3ba8576-4ee9-44a0-8785-a699e4f642d8)

One tag shall only contain one kind of api-version like:

![image](https://github.com/Azure/azure-rest-api-specs/assets/70930885/3cfaba79-013d-4873-87c8-491bd3e62532)